docs/api.md

```diff
@@ -3578,7 +3578,7 @@ Useful links:
 - Getting Started with DevTools Protocol: https://github.com/aslushnikov/getting-started-with-cdp/blob/master/README.md
 
 ```js
-const client = await page.chromium.pageTarget(page).createCDPSession();
+const client = await browser.pageTarget(page).createCDPSession();
```

Comment:
Let's keep the name `chromium` for clarity as other browsers don't have this feature.